src/wallet/rpcwallet.cpp

```diff
@@ -2575,6 +2577,258 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue bumpfee(const JSONRPCRequest& request)
+{
+    if (!EnsureWalletIsAvailable(request.fHelp))
+        return NullUniValue;
+
+    if (request.fHelp || request.params.size() < 2 || request.params.size() > 3)
+        throw runtime_error(
+                            "bumpfee \"txid\" output ( options ) \n"
```

Comment:
Maybe move output into options? Even if it's required in this version, it should ideally become optional in the future...